pkg/util/proto/openapi.go

```diff
@@ -55,6 +55,7 @@ type SchemaVisitor interface {
 	VisitMap(*Map)
 	VisitPrimitive(*Primitive)
 	VisitKind(*Kind)
+	VisitArbitraryValue(*ArbitraryValue)
```

Comment:
I'm happy you thought of that word `Arbitrary`. Can I nit and ask you to just keep `Arbitrary` without the `Value`?